ckanext/datagovuk/ckan_patches/cli.py

```diff
@@ -16,3 +16,5 @@ def set_password(ctx, username):
 
     _user = model.User.get(username)
     send_password_alert(_user)
+
+ckan.cli.user.set_password = set_password
```

Comment:
What is the significance of this line moving from plugin.py to here in ckan_patches/cli.py?

Reply:
This keeps it more in line with the other patches in the `ckan_patches` directory, and also means that it can just be imported in `plugin.py`